### DISCUSSION TITLE
CRLF vs LF settings conflict

### DISCUSSION THREAD

**Discussion by Compositr** [Elius94/console-gui-tools#70] [General]
There appears to be a conflict of settings between CRLF and LF that is causing some issues with my IDE.

The `.vscode` folder specifies VSC to use CRLF line endings (windows-style) through the `settings.json` file:

https://github.com/Elius94/console-gui-tools/blob/6e68febe7cfa364b236d48f7ec110e1fe41fd2d1/.vscode/settings.json

The ESLint settings also dictate that CRLF should be used

https://github.com/Elius94/console-gui-tools/blob/6e68febe7cfa364b236d48f7ec110e1fe41fd2d1/.eslintrc.cjs#L28

However the `.gitattributes` file specify that LF endings (UNIX-style) should be used

https://github.com/Elius94/console-gui-tools/blob/6e68febe7cfa364b236d48f7ec110e1fe41fd2d1/.gitattributes

This creates weird behaviour in my IDE and is a conflict. 

Assuming you want CRLF endings you can change the `.gitattributes` file to this to force CRLF:

```gitattributes
* text eol=crlf
```

**Comment by Compositr** [comment]
Just putting this out here in case you wanted to change it to LF. Either way it doesn't affect me, its just the conflict which is causing weird behaviour.

**Comment by Elius94** [comment]
I use windows but to have mouse in terminal I usually run this lib using WSL. So it should run properlly in both os (and others).
I think that we can change the whole project to LF because VSCode in windows work fine also with only LF

**Reply by Compositr** [reply]
Alright, made #71 which just adjusts the settings of the files and eslint.
